package.json

```diff
@@ -62,6 +63,10 @@
   },
   "homepage": "https://github.com/artsy/emission#readme",
   "files": ["index.js", "data", "lib"],
+  "resolutions": {
+    "graphql": "^0.12.3",
```

Comment:
Some of our dev deps pull in an older version, which messes with the tree we want, so just overriding it for now.